idea_inspection.bat

```diff
@@ -0,0 +1,34 @@
+@echo off
+
+::----------------------------------------------------------------------
+:: IntelliJ IDEA inspections for checkstyle.
+::----------------------------------------------------------------------
+SET PROJECT_DIR=%CD%\
+SET INSPECTIONS_PATH=%CD%\config\intellij-idea-inspections.xml
+SET RESULTS_DIR=%CD%\target\inspection-results
+SET NOISE_LVL=v1
+SET IDEA_LOCATION=
+SET IDEA_PROPERTIES=%CD%\config\idea.properties
+
+::Check IDEA_HOME env variable
+IF EXIST %IDEA_HOME% SET (
+    SET IDEA_LOCATION=%IDEA_HOME%
```

Comment:
Why can't we use `IDEA_HOME` directly?

Reply:
Because next we will try to search idea.bat in path. 
Same as below...